cpp/src/io/orc/aggregate_orc_metadata.hpp

```diff
@@ -62,7 +62,7 @@ class aggregate_orc_metadata {
 
  public:
   std::vector<metadata> per_file_metadata;
-  size_type const num_rows;
+  int64_t const num_rows;
```

Comment:
Just to make sure I understand this correctly, the total number of rows must be `int64_t` since it may exceed 2B rows while the per stripe num of rows is still using `int32_t` (or `cudf::size_type`) since we have a hard limit of the stripe size?

Reply:
Good question
That got me looking into the ORC specs:
```
 // the number of rows in the stripe
 optional uint64 numberOfRows = 5;
 ```
So in theory we could have a stripe with >2B rows. I can look into this, but maybe it's better to leave it for a separate PR.